Rakefile

```diff
@@ -6,19 +6,21 @@ end
 
 Bundler::GemHelper.install_tasks
 
-task(:default).clear
-
 require 'rubocop/rake_task'
 RuboCop::RakeTask.new
 
 require 'rspec/core'
 require 'rspec/core/rake_task'
 RSpec::Core::RakeTask.new(:spec)
 
-if ENV['APPRAISAL_INITIALIZED'] || ENV['CI']
-  task default: %i(rubocop spec)
-else
-  require 'appraisal'
-  Appraisal::Task.new
-  task default: :appraisal
+def default_task
+  if ENV['APPRAISAL_INITIALIZED'] || ENV['CI']
+    %i(rubocop spec)
+  else
+    require 'appraisal'
+    Appraisal::Task.new
+    %i(appraisal)
+  end
 end
+
+task(:default).clear.enhance(default_task)
```

Comment:
what's the motivation behind these changes?

Reply:
More linter appeasement. It wants each named `task` to only have 1 source location.